### PR TITLE
add initial python36 version stuff

### PR DESCRIPTION
**Pull request by elmiko**
* add concreate overrides for default and py36 versions
* move numpy installation to default override
* add a python36 module
* add an entry point variable
* update documentation
* remove unused script build-spark-push.sh

### CODE REVIEW

**Comment by elmiko** [comment]
specific python 3 tests to follow in another pr so that the autobuild can create an image

File: README.md [hunk 1]
```diff
@@ -4,10 +4,37 @@
...
+
+Tag and push the images to the designated reference.
+
+    make push SPARK_IMAGE=[REGISTRY_HOST[:REGISTRY_PORT]/]NAME[:TAG]
+
+# Customization
 
-    make push SPARK_IMAGE=[REGISTRY_HOST/][USERNAME]
+There are several ways to customize the construction and build process. This
+project uses the [GNU Make tool](https://www.gnu.org/software/make/) to
```
Comment (tmckayus) [inline README.md]: nit, missing a word here after "to"

File: README.md [hunk 2]
```diff
@@ -4,10 +4,37 @@
 
 # Apache Spark images for OpenShift
 
-# Build
+This repository contains several files for building
+[Apache Spark](https://spark.apache.org) focused container images, targetted
```
Comment (tmckayus) [inline README.md]: nit, no double t in targeted

File: modules/python36/module.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+schema_version: 1
+
```
Comment (tmckayus) [inline modules/python36/module.yaml]: hmm, these module.yaml files get copied into the build directories, but afaik they are used by concreate and not by the docker build. Maybe we should move them somewhere else to keep them out of the build image?  This is not new, the existing openshift-spark does this too.

@crobby am I crazy here?

**Comment by elmiko** [comment]
thanks @tmckayus updated with the grammar fixes

**Comment by tmckayus** [comment]
built both images, worked for me

**Comment by elmiko** [comment]
added a line to test the py36 images as well

**Comment by tmckayus** [comment]
merging this, I think we might have some spurious errors in travis, ran this on my local box against origin 3.9 and it passed. I'll make another PR to refactor the tests.

**Pull request merged**